src/lib/loaders/loaders_with_authentication/gravity.ts

```diff
@@ -326,6 +326,11 @@ export default (accessToken, userID, opts) => {
       ({ partnerId, inquiryId }) =>
         `partner/${partnerId}/inquiry_request/${inquiryId}/collector_profile`
     ),
+    partnerCollectorProfileLoader: gravityLoader<any, { userId: string }>(
```

Comment:
Curious what about this endpoint makes this 'partner'-ish? It looks like it's hitting a root `/api/v1/collector_profiles` endpoint with a `user_id` query param, should this just be `collectorProfileLoader`, or `collectorProfileByUserIDLoader`?

Reply:
Yes definitely! kinda just an experiment with the naming currently

Need a loader to fetch a collector profile associated with an Order (we want to show collector emails again in partner CMS)

Reply:
Also noticed this on one of @brainbicycle's PR a while back (which i think he changed). We should try to make these things as generic as possible, ideally matching them 1:1 to the gravity endpoint name.

@mzikherman - do you think we should raise this somewhere so that it doesn't become a trend across teams?

Reply:
Adjusted!